ui/app/components/sidebar/nav/cluster.js

```diff
@@ -27,6 +27,22 @@ export default class SidebarNavClusterComponent extends Component {
     return this.namespace.inRootNamespace && !this.hasChrootNamespace;
   }
 
+  get canAccessVaultUsageDashboard() {
+    /*
+    A user can access Vault Usage if they satisfy the following conditions:
+      1) They have access to sys/v1/utilization-report endpoint
+      2) They are either
+        a) enterprise cluster and root namespace
+        b) hvd cluster and /admin namespace
+    */
+
+    const hasPermission = this.permissions.hasNavPermission('monitoring');
```

Comment:
A note about the permissions helper. It relies on the [resultant-acl endpoint](https://github.com/hashicorp/vault/blob/main/ui/app/adapters/permissions.js) and that endpoint has a known bug. The short story is that if the policy has a `+` in the path it won't work. This is common for namespace situations, which _may_ (I'm guessing) affect hvd users who may do something like the following to account for admin namespaces:

```
path "+/sys/*" {
  capabilities = ["read"]
}
```

A brief look at the code and it doesn't look like we make an exception in this helper for hvd users. We can chat about this today in our sync, but it might be something to test. And hopefully, there's something I'm missing and this isn't something we need to worry about.

Reply:
Yeah - I was kind of thinking that instead of addressing this known issue in one-off scenarios we should address it more holistically and stick with the nav permission pattern as is.

Reply:
I might not know enough about policies and the known bug to understand the suggestion here, lets chat about this one we can catch up.

Reply:
As mentioned by @hellobontempo and discussed on a call with @Monkeychip this permission bug would likely need to be addressed more holistically outside the scope of this PR.